R/inverse_interpolation.R

```diff
@@ -21,3 +21,43 @@ inverse_interpolation <- function (x, y, ypred, omega) {
     return(xpred)
 }
 
+
+#' Local inverse interpolation with bandwidth selected via k-fold cross-validation.
+#'
+#' @param x The matrix of "training" values of x, with one row per training set.
+#' @param y The matrix of "training" values of y corresponding to x.
+#' @param ypred The matrix of "test" values of y for which a value of x will be predicted.
+#' @param seed The random seed used in the k-fold cross-validation.
```

Comment:
it seems strange to pass in a seed, since you can just set the seed before calling the function